e2e/.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
```

Comment:
Do we need this new file? If I'm understanding them correctly, the editorconfig docs suggest that a compliant editor should pick up the `.editorconfig` in the root of the repository - https://editorconfig.org/#file-location

Reply:
You're right. I'm new to editorconfig. I'll have to spend some time reading those docs, thanks for link. I'll remove the file.